src/MapfileStyleParser.ts

```diff
@@ -581,30 +585,30 @@ export class MapfileStyleParser implements StyleParser {
   /**
    * Get the GeoStyler-Style Symbolizers from an Mapfile CLASS.
    *
-   * @param {object} mapfileClass The Mapfile Class
+   * @param {MapfileClass} mapfileClass The Mapfile Class
    * @param {string} mapfileLayerType The Mapfile Layer Type
    * @param {string} mapfileLayerLabelItem The Mapfile Layer Label Item
    * @return {Symbolizer[]} The GeoStyler-Style Symbolizer Array
    */
   getSymbolizersFromClass(
-    mapfileClass: any,
+    mapfileClass: MapfileClass,
     mapfileLayerType: string,
     mapfileLayerLabelItem: string
   ): Symbolizer[] {
     const symbolizers = [] as Symbolizer[];
     // Mapfile STYLE
-    if (mapfileClass.style) {
-      mapfileClass.style.forEach((styleParameters: any) => {
+    if (mapfileClass.styles) {
+      mapfileClass.styles.forEach((mapfileStyle: MapfileStyle) => {
```

Comment:
```suggestion
      mapfileClass.styles.forEach((mapfileStyle) => {
```
I guess it should work without the type (implicite type)

Reply:
That is right. I removed the explicit typing.